### DISCUSSION TITLE
:Copilot  <Tab> map has been dusabled or is claimed by another plugin

### DISCUSSION THREAD

**Discussion by Sarthak950** [community/community#52918] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

![image](https://user-images.githubusercontent.com/93645760/232242383-6d930b17-465d-4d06-9721-5a84fb6cbd81.png)

how do i fix this

i use nvchad for neovim

**Comment by TeddyHuang-00** [comment, marked as answer]
I know this might be a bit too late, hope you have already got a solution. But for those who also followed Google to this discussion, there is already an solution existed on reddit: https://www.reddit.com/r/neovim/comments/wbx4r6/has_anyone_managed_to_use_github_copilot_in_nvchad/

If you do not want to follow the link, here is the quick answer: Add this line to your `.config/nvim/lua/custom/init.lua`

```lua
vim.g.copilot_assume_mapped = true
```

And it should work!

**Reply by Sarthak950** [reply]
yes i have remapped it to shift+tab that is working fine now but thanks for the suggestion I will definitely try this when I am tinkering with config again thanks
